package.json

```diff
@@ -12,22 +13,20 @@
     "lint-check": "yarn eslint src/ && yarn prettier --check src/"
   },
   "main": "dist/index.js",
-  "types": "react-native-gesture-handler.d.ts",
+  "types": "dist/index.d.ts",
```

Comment:
Should this field be still present in `package.json`? It is not present in `RNScreens`.

Reply:
AFAIK we should link to the entry file types, we can test it after PR is ready.

Reply:
This is explained in [Publishing section](https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html) of TS docs.

> Also note that if your main declaration file is named index.d.ts and lives at the root of the package (next to index.js) you do not need to mark the "types" property, though it is advisable to do so.